server/charts/openscout/values.yaml

```diff
@@ -78,7 +74,8 @@ openscoutservice:
   
 httpdservice:   
   name: httpd
-  type: NodePort
+  #type: NodePort
+  type: ClusterIP
```

Comment:
This is useful for sinfonia, but not generally. Maybe override this from our sinfonia recipe instead.